### PR TITLE
Add TS types for api, api-react and refactor parameters to object

### PR DESCRIPTION
**Pull request by zsolt-dev**

### CODE REVIEW

**Comment by paninaro** [comment]
@ChiaMineJP Can you take a look at the changes in the files related to plotting?

File: packages/api/src/wallets/NFT.ts [hunk 1]
```diff
@@ -1,86 +1,105 @@
...
-  async getNftsCount(walletId: number) {
-    return this.command('nft_count_nfts', {
-      walletId,
-    });
+  async getNftsCount(args: { walletId: number }) {
+    return this.command('nft_count_nfts', args);
   }
 
-  async getNfts(walletId: number, num: number, startIndex: number) {
+  async getNfts(args: { walletId: number; num: number; startIndex: number }) {
```
[the remark this answers is not in the document]
Comment (paninaro) [reply packages/api/src/wallets/NFT.ts:13]: The num = 100_000 is just a temporary hack until Zlatko's NFT provider is merged

File: packages/api-react/package.json [hunk 2]
```diff
@@ -9,11 +9,12 @@
...
   "main": "dist/index.js",
   "module": "dist/esm.js",
+  "types": "./dist/types",
   "scripts": {
     "build": "npm run build:js",
-    "build:js": "rollup -c",
+    "build:js": "rollup -c && tsc --emitDeclarationOnly",
     "build:ci": "npm run build:js",
-    "build:watch": "rollup -c -w",
+    "build:watch": "tsc --emitDeclarationOnly --watch & rollup -c -w",
```
Comment (ChiaMineJP) [inline packages/api-react/package.json:17]: Only `tsc --emitDeclarationOnly` part was working and never reached to `rollup`, even changing `&` to `&&`.
Maybe you need to use `concurrent` to run multiple watchers at once.

Comment (ChiaMineJP) [reply packages/api-react/package.json:17]: https://user-images.githubusercontent.com/84098616/230444399-aa72a8b1-651b-4d21-b9b5-b6d6ce967cc8.mp4

This was my experience.
Only type declaration files were created but no transpiled js files there.

Comment (ChiaMineJP) [reply packages/api-react/package.json]: Hi, I've pushed the commit which fixes Windows issue.

**Review by paninaro — APPROVED** [review]

**Pull request merged**